Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,79 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: lemonchiffon;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  background-color: white;
+}
+
+.container {
+  /* max-width: 1300px; */
+  /* margin: auto; */
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.col-2 {
+  flex-basis: 50%;
+  min-width: 300px;
+}
+
+.col-2 img {
+  max-width: 100%;
+  padding: 50px 0;
+}
+
+.col-2 h1 {
+  font-size: 40px;
+  line-height: 60px;
+  margin: 25px 0;
+}
+
+.btn {
+  display: inline-block;
+  background-color: lawngreen;
+  color: white;
+  padding: 8px 30px;
+  margin: 30px 0;
+  border-radius: 30px;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.features {
+  margin: 70px 0;
+}
+
+.feature-block {
+  flex-basis: 50%;
+  min-width: 250px;
+  margin-bottom: 30px;
+}
+
+.small-container {
+  max-width: 1080px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
+}
```

Comment:
Please refer to the answer repo to check how the flowershop website is implemented
[https://github.com/XccelerateOrg/FTWD_Jan_2022_Ans/tree/master/Web/FTWD_Web_Day_02/Exercise_D](url)